src/GraphQL/Conversion/ValueConverter.cs

```diff
@@ -1,14 +1,14 @@
 using System;
-using System.Collections.Generic;
+using System.Collections.Concurrent;
 using System.Globalization;
 using System.Numerics;
 
 namespace GraphQL
 {
     public static class ValueConverter
     {
-        private static readonly Dictionary<Type, Dictionary<Type, Func<object, object>>> _valueConversions
-            = new Dictionary<Type, Dictionary<Type, Func<object, object>>>();
+        private static readonly ConcurrentDictionary<Type, ConcurrentDictionary<Type, Func<object, object>>> _valueConversions
```

Comment:
Accidentally noticed that this static field was still not thread safe. All other static dictionaries are `ConcurrentDictionary<>`.